### PR TITLE
Fix #640 #674 Fixed issue with warnings  and Console.logs

### LINKED ISSUE #640: Remove Unwanted Console logs
**Actual Behaviour**

unwanted console.logs when ajax request returns with error.
**Expected Behaviour**

 Console should not show any console.log for objects on Production.

**Steps to reproduce it**

 Check Console while on SignUp page https://accounts.susi.ai/signup 

**Would you like to work on the issue?**

I am working on it.

### PR DESCRIPTION
**Pull request by piyushimraw**
Removed unwanted console.logs

Fixes [#640 #674] 

Changes: 
- Replaced console.logs with console.error
- Fixed error with unexpected props

Surge Deployment Link: https://pr-679-fossasia-susi-accounts.surge.sh

### CODE REVIEW

**Comment by piyushimraw** [comment]
@Akshat-Jain @akshatnitd @samagragupta @AakashMallik @PrP-11 Review and Merge

File: src/components/Auth/DeleteAccount/DeleteAccount.react.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ class DeleteAccount extends Component {
           });
         }.bind(this),
         error: function(errorThrown) {
```
Comment (AakashMallik) [inline src/components/Auth/DeleteAccount/DeleteAccount.react.js]: Rename the error var to **error** from **errorThrown**

File: src/components/Auth/Login/Login.react.js [hunk 2]
```diff
@@ -230,7 +230,7 @@ class Login extends Component {
             maxAge: time,
             domain: cookieDomain,
           });
-          console.log(errorThrown);
+          console.error(errorThrown);
```
Comment (AakashMallik) [inline src/components/Auth/Login/Login.react.js]: same.. rename to error

File: src/components/Settings/PreviewThemeChat.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ class PreviewThemeChat extends Component {
         }
       },
       error: function(e) {
-        console.log(e);
+        console.error('some erroe occured');
```
Comment (AakashMallik) [inline src/components/Settings/PreviewThemeChat.js]: Typo and rename **e** to **error** and console log the error

File: src/components/Settings/Settings.react.js [hunk 4]
```diff
@@ -358,7 +358,7 @@ class Settings extends Component {
       timeout: 3000,
       async: false,
       error: function(errorThrown) {
-        console.log(errorThrown);
+        console.error(errorThrown);
```
Comment (AakashMallik) [inline src/components/Settings/Settings.react.js]: same

File: src/components/Settings/Settings.react.js [hunk 5]
```diff
@@ -406,7 +406,7 @@ class Settings extends Component {
       timeout: 3000,
       async: false,
       error: function(errorThrown) {
-        console.log(errorThrown);
+        console.error('Some error occurred');
```
Comment (AakashMallik) [inline src/components/Settings/Settings.react.js]: same

File: src/components/Settings/Settings.react.js [hunk 6]
```diff
@@ -559,7 +559,7 @@ class Settings extends Component {
         }
       }.bind(this),
       error: function(errorThrown) {
```
Comment (AakashMallik) [inline src/components/Settings/Settings.react.js]: same

File: src/components/StaticAppBar/StaticAppBar.js [hunk 7]
```diff
@@ -64,7 +64,7 @@ class StaticAppBar extends Component {
         });
       },
       error: function(errorThrown) {
-        console.log(errorThrown);
+        console.error('some error occured');
```
Comment (AakashMallik) [inline src/components/StaticAppBar/StaticAppBar.js]: same

**Pull request merged**